docs/security-considerations.rst

```diff
@@ -62,7 +62,7 @@ complete contract):
 
     // THIS CONTRACT CONTAINS A BUG - DO NOT USE
     contract Fund {
-        /// Mapping of ether shares of the contract.
+        /// @dev Mapping of ether shares of the contract.
         mapping(address => uint) shares;
```

Comment:
These are not public, so @dev will not be useful.

Reply:
`@notice` is only for the external interface and `@dev` is valid for everything - so do you propose to default to `@dev` for non-public items?

Reply:
@axic waiting for your answer to my question.

Reply:
I think it's fine to keep the default at `@notice` for everything.